Makefile

```diff
@@ -111,10 +115,10 @@ $(RELEASE_NOTES) : $(TOOLS_DIR)/go.mod
 
 .PHONY: lint lint-full
 lint: $(GOLANGCI_LINT) ## Lint codebase
-	$(GOLANGCI_LINT) run -v
+	$(GOLANGCI_LINT) run -v --skip-files crd_manifests.go
 
 lint-full: $(GOLANGCI_LINT) ## Run slower linters to detect possible issues
-	$(GOLANGCI_LINT) run -v --fast=false
+	$(GOLANGCI_LINT) run -v --fast=false --skip-files crd_manifests.go
```

Comment:
The skip files should likely be added to the /.golangci.yml instead of on the command line here.

Reply:
moved to golangci.yml